client/public/js/components/App.js

```diff
@@ -122,6 +122,7 @@ var App = React.createClass({
                       It will be disabled when viewing the event-create and event-detail components
                     */}
                     <li>
+                      <span>{this.state.mode === 'shepherd' ? 'Manage your events' : 'Participate in events'}</span>
```

Comment:
I think you meant to reverse these two? 

this.state.mode === 'shepherd' ? 'Participate in events' : 'Manage your events'